docs/getting-started.md

```diff
@@ -1,21 +1,186 @@
+*Note: You can always use <kbd>?</kbd> to open **help** and view keyboard shortcuts.*
 
-### Narrowing to a stream
-Narrowing to a stream implies you have set middle column to show messages present in that stream. Let's try narrowing to [# test here](https://chat.zulip.org/#narrow/stream/7-test-here) from zulip terminal. Press <kbd>q</kbd> to focus on  the stream search box in the left column. Type `test here`
+## Open Zulip Terminal
+
+Once you have [installed](https://github.com/zulip/zulip-terminal#installation) the app, use `zulip-term` to run it. You will be prompted to add the URL of organization you're a part of—use chat.zulip.org to join the Zulip community server—and log into your account. If you don't already have a Zulip account, you'll have to use our [web-client](https://chat.zulip.org) to sign up because currently you can't create an account using Zulip Terminal. Learn more about how to [sign up for an account](https://zulip.com/help/logging-in).
+
+If you have issues opening Zulip Terminal, check out our [Running for the First Time](https://github.com/zulip/zulip-terminal#running-for-the-first-time) section.
+
+*Note: This tutorial assumes you're on the Zulip Community Server. While you can still follow along if you're not, various examples won't be the same (such as the [# test here](https://chat.zulip.org/#narrow/stream/7-test-here) stream).*
+
+## Understand the Layout
+<img src="https://user-images.githubusercontent.com/9568999/106061037-b659a580-60a9-11eb-8ff8-ea1c54ac9084.png" width="75%">
+
+Zulip Terminal is divided into three columns/divisions:
++ **The left column** shows list of all the streams you are subscribed to and some keyboard shortcuts for viewing different types of messages.
++ **The middle column** displays all the messages in the current narrow.
++ **The right column** shows all the active(green)/idle(yellow)/offline(white) users.
+
+## Your Cursor
+
+When you first open Zulip Terminal, your cursor will be located in the upper left-hand corner on **All Messages**. You can tell because the section will be highlighted (see image below). You can use your cursor to navigate around Zulip Terminal - let's try moving the cursor around and see what happens. Try moving it left (<kbd>left</kbd>/<kbd>h</kbd>), right (<kbd>right</kbd>/<kbd>l</kbd>), up (<kbd>Up</kbd>/<kbd>k</kbd>) and down (<kbd>Down</kbd>/<kbd>j</kbd>).
+
+<img src="getting-started-imgs/your-cursor.png" width="75%">
+
+## Keyboard Shortcuts
+
+Sometimes it can to take forever to manually move our cursor where we want it - that's why Zulip Terminal has keyboard shortcuts! Type <kbd>?</kbd> to open **help** and view a list of keyboard shortcuts. You can also view a list of [keyboard shortcuts here.](https://github.com/zulip/zulip-terminal/blob/main/docs/hotkeys.md)
+
+<img src="getting-started-imgs/help.png" width="75%">
+
+Let's try out a few of these shortcuts!
+
++ First, type <kbd>P</kbd> to view your Private Messages (PMs).
```

Comment:
There may be several other places where this applies; I see at least <kbd>S</kbd> is mentioned later in the doc.

Reply:
We don't currently standardize on this, though it has been brought up previously. This could probably be added as a point in #945, ie. to translate upper(x) to shift+lower(x).

Reply:
I changed all of the instances in this document